utils/use-google-login.tsx

```
@@ -0,0 +1,85 @@
+const provider = new GoogleAuthProvider();
+provider.addScope('profile');
+provider.addScope('email');
+// provider.addScope('https://www.googleapis.com/auth/contacts.readonly');
+// provider.setCustomParameters({
+//     'login_hint': 'user@example.com'
+// });
+
+export const useGoogleLogin = (setLoading: Function, setError: Function) => {
+    const { firebaseSignIn, firebaseSignUp } = useAuthContext();
+
+    const evalResult = async (result: void | UserCredential) => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)